target/linux/ath79/dts/qca9557_ruckus_r500.dts

```diff
@@ -0,0 +1,176 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca955x.dtsi"
+
+/ {
+	compatible = "ruckus,r500", "qca,qca9557";
+	model = "Ruckus R500";
+
+	aliases {
+		led-boot = &power;
+		led-failsafe = &power;
+		led-running = &power;
+		led-upgrade = &power;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	leds: leds {
+		compatible = "gpio-leds";
+
+		power: power {
+			label = "ruckus:red:power";
+			gpios = <&gpio 16 GPIO_ACTIVE_HIGH>;
+			default-state = "on";
+		};
+
+		dir {
+			label = "ruckus:green:dir";
```

Comment:
What's "dir"?

Reply:
DIR LED = Zone Director connection status
AIR (Signal/Air Quality) LED - Indicates uplink status and the quality of the wireless signal to the uplink AP

Added these as comments in the DTS